File: src/main/java/com/koldyr/csv/db/SQLGenerator.java
package com.koldyr.csv.db;

/**
 * Description of class SQLGenerator
 *
 * @created: 2018-09-25
 */
public interface SQLGenerator {
    String getSelectAll(String schema, String tableName);

    String getInsertValues(String schema, String tableName, String values);

    String getRowCount(String schema, String tableName);
}

File: src/main/kotlin/com/koldyr/csv/db/SQLGenerator.kt
package com.koldyr.csv.db

/**
 * Description of class SQLGenerator
 *
 * @created: 2018-09-25
 */
interface SQLGenerator {
    fun getSelectAll(schema: String, tableName: String): String

    fun getInsertValues(schema: String, tableName: String, values: String): String

    fun getRowCount(schema: String, tableName: String): String
}
